docs/ldml/tr35-keyboards.md

```diff
@@ -280,13 +280,11 @@ This is the top level element. All other elements defined below are under this e
 </keyboard>
 ```
 
-> <small>
->
-> Parents: _none_
-> Children: [version](#Element_version), [~~generation~~](#Element_generation), [info](#Element_info), [names](#Element_names), [settings](#Element_settings), [import](#Element_import), [keyMap](#Element_KeyMap), [displayMap](#Element_DisplayMap), [layer](#Element_layer), [vkeys](#Element_vkeys), [transforms](#Element_transforms), [reorders](#Element_reorder), [backspaces](#Element_backspaces)
-> Occurence: required, single
->
-> </small>
+> <small>Parents: _none_</small>
+
+> <small>Children: [version](#Element_version), [~~generation~~](#Element_generation), [info](#Element_info), [names](#Element_names), [settings](#Element_settings), [import](#Element_import), [keyMap](#Element_KeyMap), [displayMap](#Element_DisplayMap), [layer](#Element_layer), [vkeys](#Element_vkeys), [transforms](#Element_transforms), [reorders](#Element_reorder), [backspaces](#Element_backspaces)</small>
+
+> <small>Occurence: required, single</small>
```

Comment:
The old markdown generated a single <p> for the block quote, inside a <small> element, but html does not allow <p> inside <small>, only the other way around. Also the old markdown appeared to be intended to generate 3 different lines of block quote for the three different parts (Parents, Children, Occurrence), but that was not happening. So add the empty lines to generate the three separate <p> in the block quote, hopefully with the <small> inside each of the <p>s.